packages/custom-webpack/examples/append-webpack-plugins/angular.json

```diff
@@ -98,6 +124,21 @@
               "**/node_modules/**"
             ]
           }
+        },
+        "xliffmerge": {
```

Comment:
I'd really prefer to keep this example as simple as possible. As far as I understand `xliffmerge` is not necessary (as it runs after the webpack plugin was run) an it's enough to verify the file exists after simple `xi18n` run. Am I missing something? If not please remove it as it complicates the example.

Reply:
>As far as I understand xliffmerge is not necessary (as it runs after the webpack plugin was run) an it's enough to verify the file exists after simple xi18n run.

100% correct, removed as requested.